### DISCUSSION TITLE
如何选择affine数据存储的位置

### DISCUSSION THREAD

**Discussion by Angel0726** [toeverything/AFFiNE#1671] [Q&A]
affine可以将数据存储在本地，如何选择affine数据存储的位置？目前只有web使用，web如何选择数据存储位置呢

**Comment by himself65** [comment]
web端目前是存储到indexeddb上

**Comment by doodlewind** [comment]
感谢你对 AFFiNE 的关注！我们将在下一个 major release 中推出客户端，允许选择本地文件路径同步 workspace 数据。

**Comment by Angel0726** [comment]
我还想问一下affine会像appflowy一样未来提供块的插件市场吗？

**Reply by doodlewind** [reply]
是的，这是规划中的特性 :)

**Comment by Angel0726** [comment]
期待，感觉你们比appflowy开发快

**Comment by HeJiachen-PM** [comment]
Official support of on-disk storage has been added in AFFiNE 0.6.x

**Comment by lsx-xyg** [comment]
windows client 数据存储在哪里啊

**Comment by joymayson** [comment]
请问：
1、macOS 的数据存储在哪里？
2、能否修改数据存储的位置
3、能否开发批量导出功能
